### ISSUE TITLE
Style don't work / [ICO] or [DIR]

### ISSUE THREAD

**Issue by augustinpft** [AdamWhitcroft/Apaxy#39]
Hello, when i'm using apaxy on windows 8 through vagrant  and i have a little problem: apaxy won't work
![image](https://cloud.githubusercontent.com/assets/4787691/6097433/e2274e5e-afbd-11e4-86f8-cec3b62fbc2b.png)

here is the pastbin of the main htaccess: http://pastebin.com/29PZ7da9

thank you

**Comment by GGJesus** [comment]
I did exactly as the tutorial on the website said as well, but it was sensationally vague: "Download and unzip Apaxy" with no link to any sort of download, so I grabbed the GitHub ZIP and it had all the nice little things but most importantly of all it was missing an actual index file to show up and replace this default directory listing which was and is infuriating.

**Comment by GGJesus** [comment]
Ok, I believe that you must first put in http://larsjung.de/h5ai/ and then use Apaxy in the _h5ai/client/theme folder, but the installation tutorial on the website is incredibly vague as well, mentioning files that don't even exist in the download folder :(. There really should exist a video tutorial for this because it's mind boggling how bad these (link above and apaxy) tutorials are.

**Comment by AdamWhitcroft** [comment]
Sorry to hear you're struggling to get this going. Have you managed to sort it out yet?

**Comment by n3storm** [comment]
Hi GGJesus,
Installing Apaxy on your virtualhost or whole server depends on so many variables with a count as big as Apache Virtualhosts variables.
You should learn the basics of the Apache Directory and Location directives at Apache documentation. http://httpd.apache.org/docs/2.4/sections.html

For example, my setup is completely different from docs because I wanted to use the same Apaxy directory from different virtualhosts, so I used linked dirs (not ideal, better use rewrite directive).

I have downloaded zip file from http://release.larsjung.de/h5ai/h5ai-0.27.0.zip and it does contain an index.html. It is possible that your zip or tar.gz does not have this file, but you have to be exact when reporting issues to developers or projects, else is very difficult to solve the issue or give help.

Hope you take my words in a good sense :)

Cheers.

**Comment by n3storm** [comment]
@augustinpft  what does the apache log or firefox debugger network resources tab report?

**Comment by warshanks** [comment]
I'm having the same issue. I'm using Xampp if that matters.

**Comment by dexX7** [comment]
Assuming `C:\Dev\theme\icons\blank.png` maps to `http://localhost/theme/icons/blank.png` (or similar), then you may try:
```diff
...
-        AddIcon C:\Dev\theme\icons\blank.png ^^BLANKICON^^
-        AddIcon C:\Dev\theme\icons\folder.png ^^DIRECTORY^^
-        AddIcon C:\Dev\theme\icons\folder-home.png ..
+        AddIcon /theme/icons/blank.png ^^BLANKICON^^
+        AddIcon /theme/icons/folder.png ^^DIRECTORY^^
+        AddIcon /theme/icons/folder-home.png ..
...
```

Check out the source of the resulting listing, there is probably something like:
```html
<img src="C:\Dev\theme\icons\folder.png" alt="[DIR]">
```
... whereby the URI isn't valid (i.e. the image can't be reached via `C:\Dev\theme\icons\folder.png`).

**Comment by rjenkinsjr** [comment]
@augustinpft I got the same behavior as you, but I managed to fix it. My mistake was replacing `{FOLDERNAME}` with the absolute filesystem path to the theme files. Seems like it's supposed to be replaced with the *relative* path starting from your `DocumentRoot` to the `theme` directory. In my case, I was applying Apaxy to my entire `DocumentRoot`, so I replaced `{FOLDERNAME}` with nothing and it worked.

@AdamWhitcroft is this correct?
